### ISSUE TITLE
[Feature request] <Improve a feature description>

### ISSUE THREAD

**Issue by AndreiMiloiu** [notepad-plus-plus/notepad-plus-plus#15990]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description of the Issue

I think Select All In-betwen {} [] or () must be improved. What about Select All the Strings Between {}, [] or ()? Or what about Select the Text Between {}, [] or ()? Or Select the Entire Text Between {}, [] or ()?

![Feature in v8 7 5](https://github.com/user-attachments/assets/5fc367f3-1bea-4154-8360-e0f4cb9abfe9)

These are just some suggestions. You can choose an even better one.

### Describe the solution you'd like.

I think the feature description can be improved.

### Debug Information

```shell
None
```

**Comment by alankilborn** [comment]
IMO it isn't deficient enough in description to warrant a change.

**Comment by AndreiMiloiu** [comment]
Maybe to add some commas, like  {}, [] or () ?

**Comment by AndreiMiloiu** [comment]
I understand. If not, all the others except this one.

**Comment by AndreiMiloiu** [comment]
I still think a little bit better explanation will make users to better understand the usage of the feature.

**Comment by alankilborn** [comment]
>What about Select All the Strings Between {}, [] or ()? Or what about Select the Text Between {}, [] or ()? Or Select the Entire Text Between {}, [] or ()?

IMO, none of the suggestions really do anything to improve the command's menu text.  "Select All" is very common phrasing, and is preferred over "Select All the Strings" or "Select the Text" or "Select the Entire Text".

>Maybe to add some commas

Adding a comma...well, OK, but I don't think it would improve the situation all that much.  That is, probably not worth doing if that is the only change being made.

---

> I still think a little bit better explanation will make users to better understand the usage of the feature.

There's really no way to "explain" a command on its menu entry.
The place for explanation is in the user manual; see [HERE](https://npp-user-manual.org/docs/searching/#:~:text=Select%20All%20In%2Dbetween).

To be honest, I have never used this command.
So I created some text in a N++ tab:  `(foobarfubar)` and I put my caret adjacent to the `u` in that and tried to invoke the command.  It was disabled so I couldn't.  As a user, I was confused. :-(
I read the user manual; I was still confused.  :-(
Finally I figured out that for the command to become enabled, the caret has to be adjacent to a `(` or `)` character in my example text.
The user manual could be better.
I got this cue from the _Go to Matching Brace_ command description in the user manual.  That entry could also be better.

---

Suggested new text for user manual for _Go to Matching Brace_:

```
Allows parentheses and braces navigation. If the caret is in a position adjacent to the opening parenthesis ( or bracket [ or brace {, then this command will move the caret to the position before matching closing-character ) or ] or }; similarly, if the caret is in a position adjacent to the closing character, the command will move the caret to the position before the matching opening-character. (The Style Configurator > Global Styles > Brace Highlight Style will be used to highlight the opening and closing pairs of characters.)
```

Suggested new text for user manual for _Select All In-between {} [] or ()_:
```
Will select all the text in between a matching pair of {} or [] or (); the selection will include the surrounding pair of braces, brackets, or parentheses.  To invoke the command, the caret must be in a position adjacent to one of these characters.
```

**Comment by AndreiMiloiu** [comment]
I understand now. Should I close the issue?

**Comment by AndreiMiloiu** [comment]
You explained me well. However, I am still unsure about the string. It is about the types of brackets. It is an enumeration, so it should be a comma.

**Comment by alankilborn** [comment]
> so it should be a comma.

I won't quibble about a comma.

So the suggestion is to change the menu text from `Select All In-between {} [] or ()` to `Select All In-between {}, [] or ()`.

Let's ask the Notepad++ author:

@donho Do you want to change this menu text so that it adds a comma?

**Comment by AndreiMiloiu** [comment]
Yes, this is my suggestion. I will also wait @donho for an answer.
